### ISSUE TITLE
reproduce

### ISSUE THREAD

**Issue by lk0429** [zwyang6/MoRe#3]
Hello, author! Thank you for your excellent work! When I was reproducing the code, I got a result of 74.6 on the validation set, while your paper shows 76.4, which is a significant difference. Could you please help me take a look? Here are my  train.log and eval_seg_val.log.
[train.log](https://github.com/user-attachments/files/18419854/train.log)
[eval_seg_val.log](https://github.com/user-attachments/files/18419857/eval_seg_val.log)

**Comment by zwyang6** [comment]
Hi, thanks for your attention to our work! 
I notice the results from your train.log file is close to my log, but the final results after post-processings are lower. 

Have you ever tried to set the [cam2seg](https://github.com/zwyang6/MoRe/blob/d733d347e64f21425df245341e1f88900886b2bb/run_evaluate_voc.sh#L10) as True in the bash file?

**Comment by lk0429** [comment]
The command I used when evaluating VOC seg results is as follows: “python ./tools/infer_seg_voc.py --model_path /home/ac/data/lk/MoRe-main/w_outputs/2025-01/voc_train_voc_14-19-25-14/checkpoints/model_iter_20000.pth --infer_set val”. I couldn't find the parameter “cam2seg” in infer_seg_voc.py.

**Comment by zwyang6** [comment]
You may refer to ./run_evaluate_voc.sh. Changing cam2seg as True means we generate the final semantic segmentation from pseudo labels. The cls predictions are also used to achieve this process.

**Comment by lk0429** [comment]
In fact, I had already set the cam2seg parameter to true when I previously evaluated CAM and LAM. However, after re-running the run_evaluate_voc.sh script, the result remains 74.6
![image](https://github.com/user-attachments/assets/b9041f91-9afd-4f7e-a6ef-7b1c4355fdb8)

**Comment by zwyang6** [comment]
Sry. I cannot find the reason right now. 

You may run it several times to check if the results remain lower than the reported values. 
My local server enviroments is: Pytorch 1.13.1 Python 3.8 CUDA 11.7 with a RTX 3090 GPU. Also to make sure you hold the same enviroment with ours. 

I will re-run it and realse the related checkpoints as soon as possible.  Hope it is helpful to your work ^_^

**Comment by lk0429** [comment]
Thanks，I will try it.
